playaibo/views.py

```diff
@@ -0,0 +1,46 @@
+from django.shortcuts import render
+from django.views.generic import View
+import requests
+
+API_Key = 'XXXXXXXXXXXXXXXXXXXX'
```

Comment:
細かい話ですが、名前はすべて大文字にしたほうがよさそうです。

```suggestion
API_KEY = 'XXXXXXXXXXXXXXXXXXXX'
```

また、実際にリリースする際は環境変数で値を渡せるようにするのがいいと思います。